README.md

```diff
@@ -1 +1,24 @@
-# couch-tracker-documentation
\ No newline at end of file
+# Couch-Tracker
+
+An Android application, to keep track of the TV shows you follow, and the movies you've watched.
+
+This repository contains all the documents and design decisions we've made.
+
+
+## Tenets
+
+- Free and open-source
```

Comment:
Do we want to think about where the publishing will take place? Just GitHub Releases? FDroid? Play Store?

Reply:
Seems premature